NorthstarDLL/util/printmaps.cpp

```diff
@@ -182,10 +184,32 @@ void ConCommand_maps(const CCommand& args)
 			spdlog::info("({}) {}", PrintMapSource.at(map.source), map.name);
 }
 
+void ConCommand_map(const CCommand& args)
+{
+	if (args.ArgC() > 1)
+	{
+		RefreshMapList();
+
+		const char* arg = args.Arg(1);
+		auto f = [&](MapVPKInfo map) -> bool { return map.name == arg; };
```

Comment:
`f` is rly not a good variable name, i assume it stands for `filter` or something?

Reply:
I'll just put everything into one line